### ISSUE TITLE
Using the video block on non video hosting plans will return a security error

### ISSUE THREAD

**Issue by mrfoxtalbot** [Automattic/wp-calypso#32975]
When trying to use the video block in a non-video-hosting plan (Blogger / Personal) the user will see a security error:

`MVI_1513.MP4: Sorry, this file type is not permitted for security reasons.`

![](https://cld.wthms.co/Y4doqK+)
Image link: https://cld.wthms.co/Y4doqK

If we try to upload it via the Media Library we will see the correct message:

`1 file could not be uploaded because your site does not support video files. Upgrade to a premium plan for video support.`

![](https://cld.wthms.co/Mztf8j+)
Image link: https://cld.wthms.co/Mztf8j

#### Steps to reproduce
1. Starting at URL: https://wordpress.com/block-editor/post/judylou.blog (on a free, Blogger or Personal plan)
2. Add a vide block
3. Select & upload a video from your computer 

#### What I expected
I would expect a "You need to upgrade" message, like the one in the Media Library"

#### What happened instead
I got at "This file is not allowed for security reasons" instead

#### Context / Source
2024763-zen

**Comment by mmtr** [comment]
Thanks for the report @mrfoxtalbot! 

Closing this as duplicate of https://github.com/Automattic/jetpack/issues/11905

That issue is on Jetpack because the ability to upload videos is controlled by VideoPress and we are already extending/customizing the video blocks with a Jetpack extension that adds VideoPress support.

BTW, I updated the issue to better reflect that personal and blogger plans are also experiencing this issue.
